scripts/plot.py

```diff
@@ -145,3 +144,148 @@ def plot(directory_name):
         plt.grid()
         plt.savefig(directory / "GPU_Temp.png")
         plt.clf()
+
+
+def plot_comb(directory):
+    outputs, backends = load_all(directory)
+
+    plt.xlabel("Time(minutes)")
```

Comment:
```suggestion
    plt.xlabel("Time (minutes)")
```

Need a space here, but also probably a better timescale is seconds than minutes.